packages/tools/generator-pie-component/README.md

```diff
@@ -0,0 +1,80 @@
+<p align="center">
+  <img align="center" src="../../../readme_image.png" height="200" alt="">
+</p>
+
+<p align="center">
+  <a href="https://www.npmjs.com/@justeattakeaway/generator-pie-component">
+    <img alt="Currently released NPM version" src="https://img.shields.io/npm/v/@justeattakeaway/generator-pie-component.svg">
+  </a>
+</p>
+
+<div align="center">
+
+  # generator-pie-component
+
+  `generator-pie-component` is a generator for Pie Web Components.
+
+</div>
```

Comment:
Is there a particular reason for having this centering markup in this package but not in the others?